phoenix-core/src/it/java/org/apache/phoenix/end2end/ExplainPlanWithStatsEnabledIT.java

```diff
@@ -165,7 +165,7 @@ public void testBytesRowsForSelectWithLimitIgnored() throws Exception {
             ResultSet rs = conn.createStatement().executeQuery(sql);
             assertFalse(rs.next());
             Estimate info = getByteRowEstimates(conn, sql, binds);
-            assertEquals((Long) 390L, info.estimatedBytes);
+            assertEquals((Long) 691L, info.estimatedBytes);
```

Comment:
This is to undo my change on 3/17/2019. I had the impression that the Phoenix server code should never return the empty column cell back to the client. It turned out that it was not the case especially for this query. As you can see, this PR does not remove the empty column from the returned result anymore. The increase/revert is because of that.